app/routes/news._index.mdx

```diff
@@ -26,6 +26,25 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'July 30, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars are Now Available via Kafka, New Schema Release v4.1.0 and Heartbeat Kafka Topic</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        - **Stream Circulars over Kafka in JSON format** on the topic `gcn.circulars`.  Add Circulars to your existing Kafka streaming codes or generated via the [Start Streaming Interface](/quickstart).
```

Comment:
```suggestion
        - **Stream Circulars over Kafka in JSON format** on the topic `gcn.circulars`.  Add Circulars to your existing Kafka streaming codes or generated via the [Start Streaming GCN Notices interface](/quickstart).
```

Reply:
It's not just notices anymore.

Reply:
That's true, but it's still what that interface is called. The naming is inconsistent now, but that's what it's called.